build.gradle.kts

```diff
@@ -4,18 +4,14 @@ val VERSION_NAME: String by project
 val GROUP: String by project
 
 plugins {
-    kotlin("jvm") version libs.versions.kotlin.get()
+    kotlin("jvm") version libs.versions.kotlin.get() apply false
```

Comment:
Why is this not applied here? Any reason?

Reply:
The reason is simply that I was adding `kotlin("multiplatform")` plugin, and applying these two plugins together at the same time would clash, but now with the removal of the multiplatform plugin I think we can re-apply it.